src/main/java/com/lunark/lunark/reservations/service/ReservationService.java

```diff
@@ -147,6 +151,7 @@ public void acceptOrRejectReservation(Reservation reservation, ReservationStatus
             updatePropertyAvailability(reservation, true);
             updateReservations(reservation);
```

Comment:
`updateReservations` should call this method, but with `ReservationStatus.REJECT` so the guests who had their reservations rejected as a result of an overlapping one being accepted are notified.